drivers/network/dd/ne2000/netne2000.inf

```diff
@@ -64,13 +64,11 @@ LoadOrderGroup = NDIS
 [Strings]
 ; Non-localizable
 ReactOS = "ReactOS Project"
-
-RealtekMfg = "Realtek Semiconductor Co., Ltd."
+RealtekMfg = "Realtek"
```

Comment:
An additional empty line here served a purpose to separate generic and specific manufacturers. I'm ok only with "Realtek" shortening.

Reply:
see my above answer. Only one aspect should be separated by the same character. And that is Localizable vs. Non-Localizable. The newline otherwise doesn't tell anymore which aspect it ends.

Reply:
I could revert that specific newline just to satisfy your demand, but it would make things inconsistent again.
Would you approve the PR if I did that here?

Reply:
Yes, please revert superfluous deletions of empty lines in all files, then I will double check all that, and if it's okay - will approve.

Reply:
I did. The remaining ones are not superfluous. They do have a purpose, see the following picture, left side is before:
How should translators ever reach a consistent style here on the left side? We made it unnecessarily complicated for them:
See section 804 for example:
![notSuperfluous](https://user-images.githubusercontent.com/33393466/208691415-3ba96d38-690b-4c18-8a7b-7261e0c96efb.PNG)